### DISCUSSION TITLE
分组只能在文件编辑里创建吗，不能创建好组拖入吗？

### DISCUSSION THREAD

**Discussion by Geekiter** [Future-Scholars/paperlib#185] [Q&A] [answered]
因为我从zotero导出到paperlib的，所以导入的文件全部到所有文献里了，并没有保留原来的组。

我觉得如果能在组下创建好组，就算不能批量拖入，一个一个拖入也是可以的。

我看文档在创建组这块没有很明确的说明，我自己试了一下只能在文件编辑里添加组

另外还是希望能导入的时候可以指定到导入哪个组或者标签

**Comment by GeoffreyChen777** [comment, marked as answer]
paperlib设计理念，不应存在空tag folder。所以你如果想达到你描述的效果，你可以找一个论文创建folder，之后拖入其他论文。支持多选拖入。

同时也支持PDF直接拖入对应tag folder。

**Reply by Geekiter** [reply]
原来如此，谢谢

**Comment by GeoffreyChen777** [comment]
你好，3.0.0 beta 版本已经支持这个功能了。
